src/app/payment/views/IntegratedCheckoutView/CheckoutViewWrapper.tsx

```diff
@@ -340,7 +327,7 @@ const CheckoutViewWrapper = () => {
     setIsUserPaying(true);
 
     const { email, password, companyName, companyVatId } = formData;
-    const userData = getUserInfo(user, email, userNameFromAddressElement, fullName);
+    const userData = getUserInfo(email, userNameFromAddressElement, fullName, user);
```

Comment:
If the function has more than three params an object param could fit better,  induces fewer errors, especially if the function requires parameters of the same type